PlayBook/src/settings/graphics.md

```diff
@@ -8,25 +8,27 @@ This controls the Model Identifier that PlayCover will report to the running app
 
 For most applications, iPad identifiers are preffered over iPhone identifiers, however there are a few iPhone options available.
 
-M1 MacBook Airs may suffer from thermal issues when running intensive applications like Genshin Impact with the M1 iPad identifier. It is therefore recommended to use the A12Z identifier if thermals are a concern.
+Using the `M1` iPad identifier on MacBook Air will typically result in thermal throttling when playing graphically intensive games (like Genshin Impact) over a long period of time. If thermals are a concern or you want to enjoy longer gaming sessions on MacBook Air without FPS drops, we recommend using the `A12Z` identifier.
 
-For every other Apple Silicon device and most applications, we recommend the M1 iPad identifier, as this usually provides the best experience.
+For every other Apple Silicon device and most applications, we recommend the `M1` iPad identifier, as this usually provides the best experience.
 
-### Resolution
-
-1. App Default
+>__Note__: For most graphically intensive games the `A12Z` identifier will provide a consistent 60fps experience at max graphical settings, but at the cost of image sharpness since the game will be rendered at a lower internal resolution compared to `M1`.
 
-    This will leave the window size up to the application, which may not always be desirable.
-2. Auto (Based on Display)
+### Resolution
 
-    This will attempt to match the window size to your display's resolution.
-3. 1080p/1440p/4K
+<img width="295" alt="Screenshot 2023-01-11 at 2 15 52 PM" src="https://user-images.githubusercontent.com/78054566/211922033-9d0e49ac-05d5-47d5-b9db-475c8288d558.png">
 
-    This sets the window size to a standard resolution at either a 4:3, 16:9, or 16:10 aspect ratio. Most MacBook built in displays are 16:10.
-4. Custom
+| Option | Description |
+| :-----------: | ------------- |
+| App Default  | This will leave the window size up to the application, which may not always be desirable since it might not fully utilize the available screen real estate of your display. |
```

Comment:
```suggestion
| App Default  | This will leave the window size up to the application, which may not always be desirable since it might not fully utilize the available screen real estate of your display.

Some apps may only work correctly under this setting, otherwise their display and touch area might be offset, and in particular cases dynamic window resizing won’t work either. |
```